plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftCastPushdown.java

```diff
@@ -460,11 +522,35 @@ protected List<CastTestCase> unsupportedCastTypePushdown()
                 new CastTestCase("c_real", "double", "c_double_precision"),
                 new CastTestCase("c_double_precision", "real", "c_real"),
                 new CastTestCase("c_double_precision", "decimal(10,2)", "c_decimal_10_2"),
-                new CastTestCase("c_char_10", "char(50)", "c_char_50"),
-                new CastTestCase("c_char_10", "char(256)", "c_bpchar"),
-                new CastTestCase("c_varchar_10", "varchar(50)", "c_varchar_50"),
-                new CastTestCase("c_nvarchar_10", "varchar(50)", "c_nvarchar_50"),
-                new CastTestCase("c_varchar_10", "varchar(50)", "c_text"),
+
+                new CastTestCase("c_varchar_15_unicode", "char(50)", "c_char_50"),
+                new CastTestCase("c_nvarchar_15_unicode", "char(50)", "c_char_50"),
+                new CastTestCase("c_varchar_50", "char(50)", "c_char_50"),
+
+                new CastTestCase("c_char_50", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_boolean", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_smallint", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_int2", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_integer", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_int", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_int4", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_bigint", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_int8", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_real", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_float4", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_double_precision", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_float", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_float8", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_double_precision", "varchar(50)", "c_varchar_50"),
```

Comment:
`new CastTestCase("c_double_precision", "varchar(50)", "c_varchar_50")` is duplicated.